### DISCUSSION TITLE
Plugin Idea: Backlinks

### DISCUSSION THREAD

**Discussion by Jatin794** [mkdocs/mkdocs#2547] [Ideas]
The idea: It would be pretty convenient to see connections if we could add a section "Pages linking here" automatically to every page. Ideally across different base folders in docs, but even just the same section would be great

**Comment by oprypin** [comment]
Quite a nice idea. I don't foresee significant problems if someone wanted to implement it.

Appending this extra content to the bottom of the page is one way to do it. Ideally it would be integrated as a special section in themes (maybe a sidebar or something), but that would definitely require theme-specific support, which is a really hard ask for something that is not in MkDocs core, but still possible with enough momentum.

**Reply by berot3** [reply]
you are right, this would be something for @squidfunk ;)

**Reply by squidfunk** [reply]
@berot3 jup, this might make up for a nice feature 😊
